### ISSUE TITLE
Negative loss

### ISSUE THREAD

**Issue by JVD9kh96** [stwind/SketchRNN_tf2#1]
Hello, first of all I wanted to thank your for your awesome work. Then, I had a question about the training loss. After several epochs, the reconstruction loss becomes negative. Is it OK to have negative reconstruction loss? 
thanks.
